config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -29,8 +29,8 @@
         "required": "true"
     },
     {
-        "description": "Enter Object storage service provider. Supported providers are 'IBM', 'AWS' and 'Wasabi'. For other providers, you must explicitly provide 'Object Storage Service Endpoint'",
-        "displayname": "Object Storage Service Provider",
+        "description": "Available providers are 'IBM', 'AWS' and 'Wasabi'. For providers other than these, you must provide the 'Object Storage Service Endpoint' parameter.",
```

Comment:
`For providers other than these, you must provide the 'Object Storage Service Endpoint' parameter."` ---> `For providers other than these, you must provide the 'Object Storage service endpoint' parameter."`